README.md

```diff
@@ -12,3 +12,91 @@ An initial dummy implementation with some examples can be found [here](https://g
 npm ci
 npm start
 ```
+
+## Interacting with the server
+
+At this stage of development,
+the server supports low-level interaction via HTTP methods,
+such as `GET`, `PUT`, `HEAD`, ...
+
+Human-readable views are not supported yet.
+
+Below, we provide several examples on how to interact with the server using `curl`.
+
+### `PUT`: Creating resources for a given URL
+
+Create a plain text file:
+```bash
+$ curl -X PUT -H "Content-Type: text/plain" \
+  -d 'abc' \
```

Comment:
Since Unix doesn't care and Windows (of course) doesn't support `'` on the command line, I would use `"` for all examples.